### PR TITLE
Changed profiles from object to array.

### PR DESCRIPTION
**Pull request by aidanbdh**

### CODE REVIEW

File: profile.html [hunk 1]
```diff
@@ -46,5 +46,6 @@ <h3 id="profile-name"></h3>
     <script src="search.js"></script>
     <script src="friends.js"></script>
     <script src="default.js"></script>
+    <script src="createProfile.js"></script>
```
Comment (thebearingedge) [inline profile.html]: Use `lower-kebab-case` for file names.

File: createProfile.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+function CreateProfile (username) {
```
Comment (thebearingedge) [inline createProfile.js]: Where are you using this?

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: createProfile.js (hunk 2 above)
Comment (aidanbdh) [reply createProfile.js]: That should not have been committed yet my bad

**Comment by aidanbdh** [comment]
Staged commit with changes

**Pull request merged**